content/en/blog/2023/spring-native/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Observe your Spring Native image application with OpenTelemetry
+linkTitle: Spring Native
+date: 2023-11-28
+author: >-
+  [Jean Bisutti](https://github.com/jeanbisutti) (Microsoft)
+cSpell:ignore: datasource logback bisutti
+---
+
+You may know the
+[OpenTelemetry Java agent](https://opentelemetry.io/docs/instrumentation/java/automatic/).
+However, Java agents are
+[not officially supported](https://github.com/oracle/graal/issues/1065) with
+GraalVM native images today. This post presents an OpenTelemetry solution to get
+telemetry data with Spring Boot native image applications.
+
+## A history of the last months
+
+The OpenTelemetry contributors have used the existing OpenTelemetry Spring Boot
+starter to instrument Spring Boot native image applications with OpenTelemetry
+without byte code instrumentation. They have improved the starter in the last
+months. For example, they have added the OTLP log export. The OpenTelemetry
+starter provides auto-instrumentation for Spring HTTP frameworks. So, by adding
+the starter dependency to your project, you will benefit from the
+instrumentation of HTTP requests:
+
+```xml
+<dependencies>
+  <dependency>
+    <groupId>io.opentelemetry.instrumentation</groupId>
+    <artifactId>opentelemetry-spring-boot-starter</artifactId>
+    <version>1.32.0-alpha</version>
+  </dependency>
+</dependencies>
+```
```

Comment:
```suggestion
## Easy instrumention without an agent

The OpenTelemetry Spring Boot Starter allows to add OpenTelemetry to your application without byte code instrumentation. The OpenTelemetry Java contributors have used this to instrument Spring Boot Native Images. To make this experience better, they improved the Starter over the last few months: they have added the OTLP log export and an instrumentation library for Spring HTTP frameworks. 

By adding the starter dependency to your project, you will benefit from those two features out of the box:

```xml
<dependencies>
  <dependency>
    <groupId>io.opentelemetry.instrumentation</groupId>
    <artifactId>opentelemetry-spring-boot-starter</artifactId>
    <version>1.32.0-alpha</version>
  </dependency>
</dependencies>
```
```

Reply:
It has changed the meaning. I have reworded in 6fe84919b2d150c110fec3a8dae60d4cfb077ccf What do youn think?

Reply:
works for me!